assets/app/views/directives/delete-button.html

```diff
@@ -1,7 +1,10 @@
 <div class="actions action-button">
-	<i class="fa fa-trash-o" 
-	  ng-click="$event.stopPropagation(); openDeleteModal();"
-	  data-toggle="tooltip" 
-	  data-placement="left" 
-	  title="Delete"></i>
-</div>
\ No newline at end of file
+  <i class="fa fa-trash-o"
+    ng-click="$event.stopPropagation(); openDeleteModal();"
+    data-toggle="tooltip"
+    data-placement="left"
+    disable-popup-on-touch="true"
+    title="Delete"
+    aria-hidden="true"></i>
+  <a class="sr-only" href="" ng-click="openDeleteModal()">Delete {{resourceType | humanizeResourceType}} {{resourceName}}</a>
```

Comment:
Need to add the `action-button` class directly to the link or it turns the icon blue.